python/cuml/ts/stationarity.py

```diff
@@ -0,0 +1,106 @@
+import numpy as np
+
+def _is_stationary(yi: np.ndarray, pval_threshold=0.05) -> bool:
```

Comment:
I wonder if using cupy here would allow us to have a super fast to implement GPU version of this, while @teju85 issue gets picked up

Reply:
We have someone who can pick this up pretty soon. But if anyone else wants to give cupy a shot, that'd be nice too.